### DISCUSSION TITLE
Crash Override

### DISCUSSION THREAD

**Discussion by solid1goose** [codewars/codewars.com#3274] [Q&A]
Hello, I have a problem. My code doesn't pass random testing. I've been sitting for an hour trying to solve this problem, but still nothing. Please help me!
![image](https://github.com/user-attachments/assets/d1320e46-a757-47d9-9391-e7b081fd4f9a)
![image2](https://github.com/user-attachments/assets/dba0b165-9c6d-4111-b9a5-1c49cecf12a8)
![image3](https://github.com/user-attachments/assets/262faa4c-8dc8-4c70-bff1-632d53468839)

**Comment by hobovsky** [comment]
Github Discussions is not a place meant to discuss solutions to specific kata. If you have problems solving a challenge, you can ask your question in the kata discourse, or in the `#help-solve` channel of Codewars Discord.
